### ISSUE TITLE
deal with utcdate instead of localdate

### ISSUE THREAD

**Issue by 1kastner** [brandon-rhodes/pyephem#138]
Hi there,

first of all thanks for the project and sharing th code with everybody!

I have a small issue with this part:
```
def localtime(date):
    """Convert a PyEphem date into local time, returning a Python datetime."""
    microseconds = int(round(24 * 60 * 60 * 1000000 * date))
    seconds, microseconds = divmod(microseconds, 1000000)
    seconds -= 2209032000  # difference between epoch 1900 and epoch 1970
    y, m, d, H, M, S, wday, yday, isdst = _localtime(seconds)
    return _datetime(y, m, d, H, M, S, microseconds)
```

Why was localtime chosen? I prefer utc time for most of my projects and especially because the returned date is naive, when there was a switch, because of the lost `dst` information I won't know whether a time was before or after the daylight saving switch. I would favor a second method which takes the calculated seconds and computes a utc datetime object from that. Or just accepts and `tzinfo` so that the user can decide themselves.

That'd look like this:
```
def to_datetime(date, tzinfo):
    """Convert a PyEphem date into a classic datetime object"""
    microseconds = int(round(24 * 60 * 60 * 1000000 * date))
    seconds, microseconds = divmod(microseconds, 1000000)
    seconds -= 2209032000  # difference between epoch 1900 and epoch 1970
    return datetime.datetime.fromtimestamp(seconds, tzinfo)
```

Or it could be restricted that tzinfo is forced to be UTC, that'd be fine with me, but the solution presented will satisfy more needs.

If you are ok with my suggestion I will create a pull request.

**Comment by brandon-rhodes** [comment]
I think I had always just run `str(d)` to turn a date into a printable UTC date and time; I'm not sure I ever ran into a case, I suppose, where I needed to access each date component as a number. I don't, though, have any objection to a pull request (though I can't guarantee when the next version of PyEphem might come out, as my focus these days is on Skyfield, a replacement that uses NumPy arrays).

**Comment by 1kastner** [comment]
See #139

**Comment by brandon-rhodes** [comment]
It looks like this was resolved back in 2018 by #139 and #142, and I forgot to close it. Whoops! :)
